views/create.html

```diff
@@ -0,0 +1,126 @@
+{% extends 'layout.html' %}
```

Comment:
There is no sense in placing this file here without adding logic of rendering to backend. If you delete this files -> nothing will be changed.
Current `create.html` served as just static file from `public/` folder